forward_engineering/helpers/unityTagsHelper.js

```diff
@@ -112,8 +112,8 @@ const getUnityTagsFromCompMod = ({ tagsToFilter, filterBy }) => {
 	});
 };
 
-const getViewTagsStatement = ({ viewSchema, viewName }) => {
-    if (!viewSchema.unityViewTags.length) {
+const getViewTagsStatement = (viewSchema, viewName) => {
```

Comment:
but why not to update the [caller here](https://github.com/hackolade/DeltaLake/blob/main/forward_engineering/helpers/viewHelper.js#L89), instead of going away from the "property bag"